omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -93,6 +93,11 @@
 ####
 default['private_chef']['server-api-version'] = 0
 
+####
+# Inisghts
+####
+default['private_chef']['insights']['enable'] = false
+default['private_chef']['insights']['log_dir'] = '/var/log/insights'
```

Comment:
We may want to make this`/var/log/opscode/insights` to match our other services.